src/components/Breadcrumbs/Breadcrumbs.scss

```diff
@@ -13,7 +13,7 @@ $block: '.#{variables.$ns}breadcrumbs';
 
     &__item {
         flex-shrink: 1;
-        padding: 4px 8px;
+        margin: 4px 8px;
```

Comment:
This will probably break [AdaptiveTabs](https://github.com/gravity-ui/components/blob/main/src/components/AdaptiveTabs/AdaptiveTabs.tsx#L323)

Reply:
Returned padding, used outline-offset and adjusted border-radius